app/settings/page.tsx

```diff
@@ -60,6 +60,10 @@ function SettingsPage() {
     location.reload();
   };
 
+  const deleteProfileAvatar = () => {
+    //TODO
```

Comment:
This is still in TODO!

Reply:
Pushed all the fixations and complete the task

Reply:
If any feedback please let me know